x-pack/platform/plugins/shared/fleet/common/constants/routes.ts

```diff
@@ -213,6 +213,7 @@ export const DOWNLOAD_SOURCE_API_ROUTES = {
 
 export const REMOTE_SYNCED_INTEGRATIONS_API_ROUTES = {
   STATUS_PATTERN: `${API_ROOT}/remote_synced_integrations/status`,
+  INFO_PATTERN: `${API_ROOT}/remote_synced_integrations/{outputId}/remote_status`,
```

Comment:
I changed the path to remote_status to clarify that is a different status than the above. We might need to fetch the global status remotely (i.e. without outputId) and in that case we could add another endpoint like this:

`api/fleet/remote_synced_integrations/remote_status`